<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEditUnit</class>
 <widget class="QWidget" name="WidgetEditUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <widget class="QLabel" name="labelOffsetY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Offset Y [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLeaderY</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelUnitLivery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Livery:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditUnitLivery</cstring>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="spinBoxUnitHeading">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>359.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="spinBoxLeaderZ">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxLeader">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="labelUnitPositionX">
     <property name="text">
      <string>Position X [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxUnitPositionX</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelLeader">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Leader:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBoxLeader</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelUnitType">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBoxUnitType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEditUnitName"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxUnitType">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Unknown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aircraft</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Balloon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bomber (Dive)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bomber (Level)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bomber (Torpedo)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Building</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fighter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kamikaze</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warship</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBoxUnitHP">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="labelUnitPositionY">
     <property name="text">
      <string>Position Y [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxUnitPositionY</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxUnitPositionZ">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="spinBoxLeaderX">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>File:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditUnitFile</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelUnitName">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditUnitName</cstring>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxUnitSpeed">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>350.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelOffsetX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Offset X [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLeaderX</cstring>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="labelUnitSpeed">
     <property name="text">
      <string>Speed [m/s]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxUnitSpeed</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="radioButtonHostile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Hostile</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupAffiliation</string>
     </attribute>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="labelUnitRoute">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Route:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBoxUnitRoute</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelUnitAffiliation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Affiliation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="spinBoxUnitPositionY">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="20" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>339</width>
       <height>282</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QRadioButton" name="radioButtonNeutral">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Neutral</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupAffiliation</string>
     </attribute>
    </widget>
   </item>
   <item row="21" column="0" colspan="4">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="spinBoxUnitPositionX">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QComboBox" name="comboBoxUnitRoute">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="spinBoxLeaderY">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Hit Points:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxUnitHP</cstring>
     </property>
    </widget>
   </item>
   <item row="17" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>kts</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBoxWingman">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Wingman</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QCheckBox" name="checkBoxUnitRoute">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Route</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1" colspan="3">
    <widget class="QCheckBox" name="checkBoxOwnship">
     <property name="text">
      <string>Ownship</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="labelUnitPositionZ">
     <property name="text">
      <string>Position Z [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxUnitPositionZ</cstring>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="labelUnitHeading">
     <property name="text">
      <string>Heading [deg]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxUnitHeading</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QDoubleSpinBox" name="spinBoxUnitPositionZFt">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButtonUnitFile">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditUnitFile">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QDoubleSpinBox" name="spinBoxUnitSpeedKts">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="radioButtonFriend">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Friend</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupAffiliation</string>
     </attribute>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="labelOffsetZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Offset Z [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLeaderZ</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditUnitLivery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonUnitLivery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBoxUnique">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Unique</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditUnitName</tabstop>
  <tabstop>radioButtonFriend</tabstop>
  <tabstop>radioButtonHostile</tabstop>
  <tabstop>radioButtonNeutral</tabstop>
  <tabstop>comboBoxUnitType</tabstop>
  <tabstop>lineEditUnitFile</tabstop>
  <tabstop>pushButtonUnitFile</tabstop>
  <tabstop>spinBoxUnitHP</tabstop>
  <tabstop>lineEditUnitLivery</tabstop>
  <tabstop>pushButtonUnitLivery</tabstop>
  <tabstop>checkBoxWingman</tabstop>
  <tabstop>comboBoxLeader</tabstop>
  <tabstop>spinBoxLeaderX</tabstop>
  <tabstop>spinBoxLeaderY</tabstop>
  <tabstop>spinBoxLeaderZ</tabstop>
  <tabstop>checkBoxUnitRoute</tabstop>
  <tabstop>comboBoxUnitRoute</tabstop>
  <tabstop>spinBoxUnitPositionX</tabstop>
  <tabstop>spinBoxUnitPositionY</tabstop>
  <tabstop>spinBoxUnitPositionZ</tabstop>
  <tabstop>spinBoxUnitSpeed</tabstop>
  <tabstop>spinBoxUnitHeading</tabstop>
  <tabstop>checkBoxOwnship</tabstop>
  <tabstop>pushButtonDone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupAffiliation"/>
 </buttongroups>
</ui>
